metadata_backend/helpers/validator.py

```diff
@@ -2,15 +2,16 @@
 
 import re
 from io import StringIO
-from typing import Any, Dict, cast
+from typing import Any, Dict
 from urllib.error import URLError
 
 import ujson
 from aiohttp import web
+from defusedxml.ElementTree import ParseError
```

Comment:
the same with this: https://github.com/sissaschool/xmlschema/blob/master/xmlschema/validators/schemas.py#L29